.pre-commit-config.yaml

```diff
@@ -25,5 +25,6 @@ repos:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: detect-aws-credentials
+        args: [--allow-missing-credentials]
```

Comment:
Needed to add this in because I was getting this error trying to commit:
```
No AWS keys were found in the configured credential files and environment variables.
Please ensure you have the correct setting for --credentials-file
```

Reply:
I'd double-check that you're on the latest version of the CLI. We have this argument as part of the bundle template. 🙏 

https://github.com/massdriver-cloud/massdriver-cli/blob/main/pkg/bundle/templates/terraform/.pre-commit-config.yaml#L50

Reply:
I'm not, and no matter which way I try to update, it keeps telling me I have a newer version of the CLI available. We can continue this thread on slack 👍